consensus/src/consensus_provider.rs

```diff
@@ -48,6 +50,8 @@ pub fn start_consensus(
     ));
     let time_service = Arc::new(ClockTimeService::new(runtime.handle().clone()));
 
+
+    // REVIEW: what bugs could arise if we hit the backpressure of 1024 elements? Is a deadlock possible?
```

Comment:
I don't think deadlock is possible, the sender is on the different thread dedicated to send. although I think it's good to have a LIFO queue with size 1 for this.

Reply:
I see. With a LifoQueue of 1 you're basically treating this like a condition variable (i.e., you can tolerate one wakeup for multiple signals)

Reply:
I miss the #filetask feature we have, so we can file task easily and not risk forgetting during the auditing.